packages/cli/src/lingui-extract.ts

```diff
@@ -51,16 +54,19 @@ export default function command(
     console.log()
   })
 
-  console.error(
-    `(use "${chalk.yellow(
-      helpRun("extract")
-    )}" to update catalogs with new messages)`
-  )
-  console.error(
-    `(use "${chalk.yellow(
-      helpRun("compile")
-    )}" to compile catalogs for production)`
-  )
+  if (!options.watch) {
```

Comment:
hm i don't get this logic, why should we log this if watch isn't enabled?

Reply:
This is logged always after executing `extract` command. Not my code, not by my assumption. I only added a flag to disable this when the watcher is enabled, as it is illogical to show this in my opinion on extract. But my PR is not about this, so I have left is as is.